packages/@aws-cdk/aws-stepfunctions-tasks/lib/emr/emr-create-cluster.ts

```diff
@@ -356,6 +383,22 @@ export class EmrCreateCluster extends sfn.TaskStateBase {
 
     return role;
   }
+
+  /**
+   * Validates the release label string is in proper format.
```

Comment:
I would recommend adding a link to https://docs.aws.amazon.com/emr/latest/ReleaseGuide/emr-release-components.html and copying the statement from that page:
> Release labels are in the form `emr-x.x.x`. For example, `emr-5.33.0`.